addOns/sqliplugin/src/test/java/org/zaproxy/zap/extension/sqliplugin/ActiveScannerTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.sqliplugin;
+
+import org.parosproxy.paros.core.scanner.AbstractPlugin;
+import org.zaproxy.zap.testutils.ActiveScannerTestUtils;
+
+abstract class ActiveScannerTest<T extends AbstractPlugin> extends ActiveScannerTestUtils<T> {
```

Comment:
This should be done in `SQLInjectionScanRuleUnitTest`, there's no other rule here.